sphinx/registry.py

```diff
@@ -332,8 +332,6 @@ def load_extension(self, app, extname):
 
         if metadata is None:
             metadata = {}
-            if extname == 'rst2pdf.pdfbuilder':
-                metadata['parallel_read_safe'] = True
```

Comment:
With this removal, `rst2pdf` will be warned "the rst2pdf extension does not declare if it is safe for parallel reading" if users use `-j` option.